### PR TITLE
fix(veeam-backup): remove sorting and org fetching per column

### PR DESCRIPTION
**Pull request by chipp972**
ref: MANAGER-14884

| Question         | Answer
| ---------------- | ---
| Branch?          | `develop` 
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | MANAGER-14884 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

### CODE REVIEW

**Review by anooparveti — CHANGES_REQUESTED** [review]
Can you please separate the changes in `manager-components` into a different PR?

File: packages/manager/apps/veeam-backup/src/pages/listing/DatagridCell.component.tsx
Comment (anooparveti) [inline packages/manager/apps/veeam-backup/src/pages/listing/DatagridCell.component.tsx:1]: Can you fix the lint errors?

**Review by anooparveti — CHANGES_REQUESTED** [review]

**Comment by chipp972** [comment]
> Can you please separate the changes in `manager-components` into a different PR?

Actually I am reverting manager-components modifications from the epic branch

File: packages/manager/apps/veeam-backup/src/data/api/vcd-organization.api.ts
Comment (anooparveti) [inline packages/manager/apps/veeam-backup/src/data/api/vcd-organization.api.ts:1]: No need to append `.api` in the file name.

Comment (chipp972) [reply packages/manager/apps/veeam-backup/src/data/api/vcd-organization.api.ts:1]: fixed

**Review by darsene — APPROVED** [review]

**Pull request merged**